presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -85,6 +86,15 @@ default boolean schemaExists(ConnectorSession session, String schemaName)
      */
     ConnectorTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName);
 
+    /**
+     * Returns an error for connectors which do not support table version AS OF expression.
+     * Only iceberg connector supports table version based on AS OF expression.
```

Comment:
Hopefully this isn't true in the future--let's just remove this line, it will get out of date